sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeDirectoryClient.cs

```diff
@@ -1701,11 +1754,14 @@ public virtual DataLakeDirectoryClient GetSubDirectoryClient(string subdirectory
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable AZC0002
```

Comment:
I think the disable AZC0002 and enabled can be removed.

Reply:
unfortunately it complains :/ so we'll have to keep this one.